ra/deployment/src/main/java/io/quarkus/artemis/jms/ra/deployment/ArtemisResourceAdapterProcessor.java

```diff
@@ -0,0 +1,67 @@
+package io.quarkus.artemis.jms.ra.deployment;
+
+import java.util.Collection;
+import java.util.stream.Stream;
+
+import org.apache.activemq.artemis.api.core.client.loadbalance.ConnectionLoadBalancingPolicy;
+import org.apache.activemq.artemis.core.remoting.impl.netty.NettyConnectorFactory;
+import org.apache.activemq.artemis.ra.ActiveMQResourceAdapter;
+import org.apache.activemq.artemis.spi.core.remoting.ClientProtocolManagerFactory;
+import org.jboss.jandex.ClassInfo;
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.IndexView;
+
+import io.netty.channel.epoll.EpollSocketChannel;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.IndexDependencyBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.RuntimeInitializedClassBuildItem;
+
+public class ArtemisResourceAdapterProcessor {
+    private static final String FEATURE = "artemis-jms-ra";
+
+    @SuppressWarnings("unused")
```

Comment:
This shouldn't be necessary (at least if using IDEA and the latest Quatkus Tools plugin)

Reply:
Fix